packages/ui-proxy/src/constants.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2011-2018 Genestack Limited
+ * All Rights Reserved
+ * THIS IS UNPUBLISHED PROPRIETARY SOURCE CODE OF GENESTACK LIMITED
```

Comment:
This is, in fact, published under MIT license open source code. See [correct notice e.g. here](https://github.com/genestack/JS-SDK/blob/master/packages/system-api/src/application.ts#L1-L6).